src/js_parser.zig

```diff
@@ -21226,14 +21226,72 @@ fn NewParser_(
                         },
                         logger.Loc.Empty,
                     );
+                    var call_args = all_call_args[1..];
+                    var bind_args = all_call_args[0..1];
+                    bind_args[0] = this_module;
+                    const get_require = p.newExpr(
+                        E.Dot{
+                            .name = "require",
+                            .target = this_module,
+                            .name_loc = logger.Loc.Empty,
+                        },
+                        logger.Loc.Empty,
+                    );
+
```

Comment:
now that the cause of the GC crash is clear we could probably move .bind() back to C++